kink/inject.py

```diff
@@ -203,4 +205,12 @@ def _decorator(_service: ServiceDefinition) -> ServiceResult:
     return _decorator(_service)
 
 
-__all__ = ["inject"]
+class _Injectable:
+    def __getitem__(self, item: Type[I]) -> I:
```

Comment:
I am not really happy about it, but tried `__class_getitem__` and generic class, but IDE is not able to recognize it properly. mypy is throwing errors, and pylint is exploding :D. So it is what it is.